cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/offers/Offer.java

```diff
@@ -1,16 +1,16 @@
-package uk.gov.hmcts.cmc.claimstore.models.offers;
+package uk.gov.hmcts.cmc.domain.models.offers;
 
 import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
 import org.hibernate.validator.constraints.NotBlank;
-import uk.gov.hmcts.cmc.claimstore.constraints.FutureDate;
+import uk.gov.hmcts.cmc.domain.models.constraints.FutureDate;
+import uk.gov.hmcts.cmc.domain.utils.ToStringStyle;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'uk.gov.hmcts.cmc.domain.utils.ToStringStyle'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)